### ISSUE TITLE
reproduce acc 73.5%

### ISSUE THREAD

**Issue by hhqweasd** [Chuhanxx/Temporal_Query_Networks#6]
I trained tqn on Diving48 with 4 v100 gpus and a batchsize of 16, the best acc output by your test code on 1st stage is about 60%, and the final acc on 2nd stage is 73.5%, I guess this acc is per video acc. Then I used my code to compute per class acc, got about 65% on the second stage. 
Both results are lower than the reported results(74.5% per class acc, 80+% per video acc).

**Comment by Chuhanxx** [comment]
Hi, 
Are you using the annotations from Diving48 v2 instead of v1? The original version is noisy and lower results are expected.
If the version is correct, could you please share the training and val curves from tensorboard to help troubleshooting?

**Comment by hhqweasd** [comment]
Yes, Diving48 v2. My platform doesn't support tensorboard. 
 
I'm checking the configs in 2nd stage: **--pretrained_weights_path** and **--resume**
currently, I'm using:
--pretrained_weights_path', default='../exps/xxx/best.pth'
--resume', default=-1

**Comment by hhqweasd** [comment]
Could you provide evaluation codes for computing the missing index (per cls accuracy or per video accuracy), and the detailed and complete configuration (with a specific file path, epoch, resume, pertained-weights-path, lr) of Stages I and II on your machine or the best checkpoints? These files may help me a lot, thanks!
